builtin/logical/pki/ca_util.go

```diff
@@ -29,20 +29,21 @@ func (b *backend) getGenerationParams(
 	}
 
 	role = &roleEntry{
-		TTL:              time.Duration(data.Get("ttl").(int)) * time.Second,
-		KeyType:          data.Get("key_type").(string),
-		KeyBits:          data.Get("key_bits").(int),
-		AllowLocalhost:   true,
-		AllowAnyName:     true,
-		AllowIPSANs:      true,
-		EnforceHostnames: false,
-		OU:               data.Get("ou").([]string),
-		Organization:     data.Get("organization").([]string),
-		Country:          data.Get("country").([]string),
-		Locality:         data.Get("locality").([]string),
-		Province:         data.Get("province").([]string),
-		StreetAddress:    data.Get("street_address").([]string),
-		PostalCode:       data.Get("postal_code").([]string),
+		TTL:                  time.Duration(data.Get("ttl").(int)) * time.Second,
+		KeyType:              data.Get("key_type").(string),
+		KeyBits:              data.Get("key_bits").(int),
+		AllowLocalhost:       true,
+		AllowAnyName:         true,
+		AllowIPSANs:          true,
+		EnforceHostnames:     false,
+		AllowedSerialNumbers: []string{"*"},
```

Comment:
This should not be set to allow any by default. The user should need to explicitly add that. (There are two other places this needs to be changed). Also it doesn't actually read `allowed_serial_numbers` :-/

Reply:
Ah, never mind. I thought this was in path_roles, I skipped around too much.